test/testdata/rewriter/rails/delegate.rb.rewrite-tree.exp

```diff
@@ -169,7 +169,7 @@ class <emptyTree><<C <root>>> < (::<todo sym>)
 
     <self>.delegate(:foo, :to, :thing, :prefix, local)
 
-    <self>.delegate(:foo, local, :thing)
+    <self>.delegate(:foo, {local => :thing})
```

Comment:
This is a good example of what's fixed by this PR: the `local => :thing` should never have been treated as keyword arguments.